File: javascript/test/Question1.test.js
const chai = require('chai');
const assert = chai.assert;
const dayOfProgrammer = require('../question1')

describe('Question 1', () => {
  it ('Can get the correct date for test case 1', done => {
    let result = dayOfProgrammer(2019, 256)
    assert.equal(result, '13.09.2019')
    done()
  })

  it ('Can get the correct date for test case 2', done => {
    let result = dayOfProgrammer(2020, 128)
    assert.equal(result, '07.05.2020')
    done()
  })
})

File: injava/test/Question1Test.java
package injava.test;

import static org.junit.Assert.*;
import org.junit.Before;
import org.junit.Test;
import injava.Question1;

public class Question1Test {
    
    private Question1 question;

    @Before
    public void before() {
        this.question = new Question1();
    }
    
    @Test
    public void test1 () {
        String result = this.question.dayOfProgrammer(2019, 256);
        assertEquals("13.09.2019", result);
    }

    @Test
    public void test2 () {
        String result = this.question.dayOfProgrammer(2020, 128);
        assertEquals("07.05.2020", result);
    }
}